src/gui/MainWindowAdaptor.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file was generated by qdbusxml2cpp version 0.8
+ * Command line was: qdbusxml2cpp -c MainWindowAdaptor -a MainWindowAdaptor.h:MainWindowAdaptor.cpp org.keepassxc.MainWindow.xml
+ *
+ * qdbusxml2cpp is Copyright (C) 2015 The Qt Company Ltd.
+ *
+ * This is an auto-generated file.
+ * Do not edit! All changes made to it will be lost.
+ */
+
+#include "MainWindowAdaptor.h"
+#include <QtCore/QMetaObject>
+#include <QtCore/QByteArray>
+#include <QtCore/QList>
+#include <QtCore/QMap>
+#include <QtCore/QString>
+#include <QtCore/QStringList>
+#include <QtCore/QVariant>
+
+/*
+ * Implementation of adaptor class MainWindowAdaptor
+ */
+
+MainWindowAdaptor::MainWindowAdaptor(QObject *parent)
+    : QDBusAbstractAdaptor(parent)
+{
+    // constructor
+    setAutoRelaySignals(true);
+}
+
+MainWindowAdaptor::~MainWindowAdaptor()
+{
+    // destructor
+}
+
+void MainWindowAdaptor::appExit()
+{
+    // handle method call org.keepassxc.MainWindow.appExit
```

Comment:
All these comments should be doc comments above the method signature.

Reply:
This file is auto-generated, the next time it will be generated it will have same comment as now

Reply:
I know, but it's so short, I don't even know if it's worth maintaining an XML file for it.